azavea/cac-tripplanner
# Bugfix/kak/fix invalidate size uninitialized map

## Overview

Fixes issue with mobile site not loading directions if the map had not yet been loaded.

Also includes minor updates to some JS dependencies.


## Testing Instructions

 * (Re-)load mobile site home page
 * Should be able to get directions from home page form/directions button on place cards
 * Map tiles should still load consistently on Chrome

Fixes #1247
